src/Campaigns/resources/admin/components/MergeCampaign/Modal/index.tsx

```diff
@@ -0,0 +1,90 @@
+import {useEffect, useState} from 'react';
+import {__} from '@wordpress/i18n';
+import MergeCampaignsForm from './../Form';
+
+/**
+ * Remove the "action" query parameter from the current URL
+ *
+ * @unreleased
+ */
+const removeActionParam = () => {
+    const queryParams = new URLSearchParams(window.location.search);
```

Comment:
If we do that, the modal to merge campaigns isn't opened.

Reply:
Hmmm, that is also strange. If it doesn't work that way, rollback to what we had before.

Reply:
Yeah, let's keep it this way because it's working. As a note, I used the same approach in the tabs navigation functions (from the campaign details page) and it was working. But in a subsequent PR you moved it to the top of the file as you suggested here and the tabs navigation stop to work, so I had to roll back to this same approach there as well.